WordCounter/src/main/java/edu/utoledo/nlowe/WordCount/WordCounters/FrontSelfAdjustingWordCounter.java

```diff
@@ -30,6 +30,7 @@ public void encounter(String word)
         }
         else if (head.getValue().getValue().equals(word))
         {
+            comparisons++;
```

Comment:
This is in the wrong place. A comparison still takes place, even if it evaluates to false. Will fix in the next commit